presto-main/src/main/java/com/facebook/presto/operator/OperatorContext.java

```diff
@@ -409,24 +413,41 @@ public synchronized long requestMemoryRevoking()
     {
         boolean alreadyRequested = isMemoryRevokingRequested();
         if (!alreadyRequested && revocableMemoryReservation > 0) {
```

Comment:
Consider inlining the `alreadyRequested` variable and call to `isMemoryRevokingRequested()`